pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -204,17 +204,44 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 		return perrors.Wrap(err, "Error reconciling metrics service")
 	}
 
+	// Get the appropriate current scale from the metric, and right size
+	// the scaleTargetRef based on it.
+	want, err := c.desiredScale(ctx, pa)
+	if err != nil {
+		return err
+	}
+
+	// Compare the desired and observed resources to determine our situation.
+	got, err := c.addressCount(ctx, pa)
```

Comment:
The `addressCount` function is now superfluous. I missed this instance of "get-endpoint-and-count-addresses" code.

You can instead import `resourceutil github.com/knative/serving/pkg/resources` and use

```golang
got, err := resourceutil.FetchReadyAddressCount(endpointsLister, pa.Namespace, pa.Spec.ServiceName)
```

Reply:
Done.